valve-monitor/web-app/src/services/DomainModel.ts

```diff
@@ -34,7 +34,7 @@ export interface Event {
 
 export interface DeviceEnvVars {
   readonly deviceID: DeviceID["deviceUID"];
-  environment_variables?: object;
+  environment_variables: any;
```

Comment:
Was this necessary? The environment variables should always be an object, right?

Reply:
You're right. Changed back